$NetBSD$

One needs to explicitly enable building in the toolbar code, yet
this turns it on by default, which not everyone wants.

Let's attempt to convince xterm of this here.

--- main.c.orig	2009-01-24 17:08:01.000000000 +0100
+++ main.c
@@ -881,7 +881,7 @@ static XtResource application_resources[
     Bres("sessionMgt", "SessionMgt", sessionMgt, True),
 #endif
 #if OPT_TOOLBAR
-    Bres(XtNtoolBar, XtCToolBar, toolBar, True),
+    Bres(XtNtoolBar, XtCToolBar, toolBar, False),
 #endif
 #if OPT_MAXIMIZE
     Bres(XtNmaximized, XtCMaximized, maximized, False),
